build/build_docker.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -ex
 
-nopush=${NOPUSH:-"false"}
+nopush=${NOPUSH:-"true"}
```

Comment:
nit: I think this was called "nopush" because the default is false, and a true state indicated an exception to the default. 

Let's just avoid the double negative here, and just renaming this variable "PUSH" instead? Then it's a lot clearer when something needs this to be pushed rather than passing the rather confusing `NOPUSH=false`